Linear_cell_complex/benchmark/Linear_cell_complex_2/surface_mesh/IO_off.cpp

```diff
@@ -24,7 +24,8 @@
 
 #include "IO.h"
 #include <stdio.h>
-
+#include <CGAL/boost/graph/named_function_params.h>
+#include <CGAL/boost/graph/named_params_helper.h>
 
```

Comment:
@lrineau The Solver_interface dependency comes named_params_helper, which includes CGAL/Default_diagonalize_traits.h, a header from Solver_interface

Reply:
@sgiraudot why?

Reply:
Because one named parameter (`diagonalize_traits`) has default values that are in this header.

Reply:
For both `GetDiagonalizeTraits` and `GetSvdTraits`, forward-declarations of:
- the class template `Default_diagonalize_traits`,
- and of `Eigen_svd` and `Lapack_svd`
should be sufficient for the file `BGL/include/CGAL/boost/graph/named_params_helper.h`.

That would remove that strange dependency.

I also wonder why `GetDiagonalizeTraits` and `GetSvdTraits` are defined in that file `BGL/include/CGAL/boost/graph/named_params_helper.h`, whereas they are used only in PSP.

Reply:
@afabri @sgiraudot Do we want to remove that dependency on Solver_traits or not? It seems artificial.